Loop/Managers/DeviceDataManager.swift

```diff
@@ -147,6 +150,32 @@ final class DeviceDataManager {
 
         return Manager.init(rawState: rawState) as? PumpManagerUI
     }
+    
+    var availableCGMManagers: [AvailableDevice] {
+        return pluginManager.availableCGMManagers + availableStaticCGMManagers
+    }
+    
+    public func cgmManagerTypeByIdentifier(_ identifier: String) -> CGMManagerUI.Type? {
+        return pluginManager.getCGMManagerTypeByIdentifier(identifier) ?? staticCGMManagersByIdentifier[identifier] as? CGMManagerUI.Type
+    }
+    
+    private func cgmManagerTypeFromRawValue(_ rawValue: [String: Any]) -> CGMManager.Type? {
+        guard let managerIdentifier = rawValue["managerIdentifier"] as? String else {
+            return nil
+        }
+        
+        return cgmManagerTypeByIdentifier(managerIdentifier)
+    }
+    
+    func cgmManagerFromRawValue(_ rawValue: [String: Any]) -> CGMManagerUI? {
+        guard let rawState = rawValue["state"] as? CGMManager.RawStateValue,
+            let Manager = cgmManagerTypeFromRawValue(rawValue)
```

Comment:
[nit] `Manager` is a variable of type `Type`. Should it be capitalized or not?

Reply:
I'm following the pattern for the pump managers. See line 164.